### PR TITLE
llama: move suppress_tokens handling to common/sampling

### PR DESCRIPTION
**Pull request by ngxson**
## Overview

Move the `suppress_tokens` logits bias handling to common, so that it can be reused by other models (for example, https://github.com/ggml-org/llama.cpp/pull/26254)

## Requirements

- I have read and agree with the [contributing guidelines](https://github.com/ggml-org/llama.cpp/blob/master/CONTRIBUTING.md)
- AI usage disclosure: yes, pi with glm-5.2

### CODE REVIEW

**Review by ggerganov — APPROVED** [review]
The `common_params_sampling::has_logit_bias()` method can now be removed.
